templates/projects/non_stats.html

```diff
@@ -9,8 +9,14 @@
         {{helper.render_project_local_nav(project, 'stats', current_user, pro_features, upload_method)}}
     </div>
     <div class="col-md-9" style="margin-top:50px;">
+        <div class="row">
         {{helper.render_project_title(project, upload_method, subtitle='Statistics') }}
-        <span class="alert alert-warning"><strong>Ooops</strong> Sorry, the project does not have tasks to process and show some statistics</span>
+        <span class="alert alert-warning"><strong>Ooops</strong> Sorry, the project does not have tasks to process and
+            show some statistics</span>
+        </div>
+        <div class="row" style="margin-top: 2em;" id="grafana-link">
+            <a class="btn btn-primary" target="_blank" href="https://grafana.prod.bloomberg.com/d/f59623f5-0ab2-4849-80d3-f4ef8bda7259/poc-crowd-task-metrics?var-project_id={{project.id}}&var-task_source=GIGWORK_{{'PRIV' if private_instance else 'PUBLIC'}}">Grafana Metrics Dashboard</a>
```

Comment:
This url should be moved into `settings_local.py` and populated similar to the following:

```html
{% if config.get('DASHBOARD_URL') %}
   {% dashboard_url = config.get('DASHBOARD_URL') + "var-project_id=" + project.id + "&var-task_source=GIGWORK_" + ("PRIV" if private_instance else "PUBLIC") %}
   <div class="row" style="margin-top: 2em;" id="grafana-link">
      <a class="btn btn-primary" target="_blank" href="{{ dashboard_url }}">Metrics Dashboard</a>
   </div>
{% endif %}
```

Reply:
updated via 5537332